project/build.sbt

```diff
@@ -1,15 +1,13 @@
 
 val scalaJSVersion =
-  Option(System.getenv("SCALAJS_VERSION")).getOrElse("0.6.18")
+  Option(System.getenv("SCALAJS_VERSION")).getOrElse("0.6.25")
 
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion)
 
 addSbtPlugin("com.eed3si9n" % "sbt-doge" % "0.1.5")
 addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.8.0")
-addSbtPlugin({
-  if (scalaJSVersion.startsWith("0.6."))
-    "org.scala-native" % "sbt-scalajs-crossproject" % "0.2.0"
-  else
-    "org.scala-native" % "sbt-crossproject" % "0.2.0"
-})
-addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.1")
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "0.6.0")
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.8")
+addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "0.6.0")
+
+addSbtCoursier
```

Comment:
woah, I didn't know about this 😍

Reply:
I should mention it in the coursier README…